src/oc/electron/main.cljs

```diff
@@ -54,6 +54,8 @@
                       win-frame-settings
                       {:width w
                        :height h
+                       :minWidth 1280
```

Comment:
Isn't a bit too much 1280 as minimum width? If you have a mac 13 inch it should be good but what about all the smaller laptop?
I think we should set it to 1024 for example, WYT?

Reply:
@bago2k4 originally I did set it to 1024, because that felt like a natural number, but the responsive view didn't look quite right at that size. It created a horizontal scroll bar. I can change it back for you to see it.